src/aio.rs

```diff
@@ -87,13 +125,21 @@ pub(crate) enum ActualConnection {
     /// Represents a Tokio TCP connection.
     #[cfg(feature = "tokio-comp")]
     TcpTokio(TcpStreamTokio),
+    /// Represents a Tokio TLS encrypted TCP connection
+    #[cfg(feature = "tokio-comp")]
+    #[cfg(feature = "tls")]
```

Comment:
Can we combine these feature selects into a single line? I assume this would be:

```suggestion
    #[cfg(any(feature = "tokio-comp", feature = "tls"))]
```

Reply:
Wait, isn't this what we have the `tokio-tls-comp` feature for? Gosh, we're getting too many features :D

Reply:
Guess that applies to all usage of similar things below, also for `async-std-tls-comp`